elements/bulbs-video/components/revealed.js

```diff
@@ -269,6 +245,14 @@ export default class Revealed extends React.Component {
     GoogleAnalytics.init(player, videoMeta.gaPrefix);
     Comscore.init(player, global.BULBS_ELEMENTS_COMSCORE_ID, videoMeta.player_options.comscore.metadata);
 
+    if (this.props.onPlay) {
```

Comment:
Yes, but that is not using the same semantics as our clicktracking system, might need to coordinate with Rachel/Kelly to see if they can re-use that tracking.